<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>570</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_8">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QListWidget" name="listWidget"/>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_7">
        <property name="spacing">
         <number>20</number>
        </property>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <property name="spacing">
           <number>12</number>
          </property>
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>ВУЗ</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="uni">
            <item>
             <property name="text">
              <string>ВолгГТУ</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>ВолГУ</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <property name="spacing">
           <number>12</number>
          </property>
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Наименование специальности</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="spec"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <property name="spacing">
           <number>12</number>
          </property>
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Конкурс прошлого года</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="compitition">
            <property name="decimals">
             <number>1</number>
            </property>
            <property name="maximum">
             <double>15.000000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <property name="spacing">
           <number>12</number>
          </property>
          <item>
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Размер оплаты при договорном обучении</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="fee">
            <property name="minimum">
             <number>10000</number>
            </property>
            <property name="maximum">
             <number>30000</number>
            </property>
            <property name="singleStep">
             <number>500</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QCheckBox" name="master">
          <property name="text">
           <string>Наличие магистратуры</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <property name="spacing">
           <number>12</number>
          </property>
          <item>
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Дата начала подачи документов</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDateEdit" name="startDate">
            <property name="dateTime">
             <datetime>
              <hour>0</hour>
              <minute>0</minute>
              <second>0</second>
              <year>2024</year>
              <month>1</month>
              <day>1</day>
             </datetime>
            </property>
            <property name="maximumDateTime">
             <datetime>
              <hour>23</hour>
              <minute>59</minute>
              <second>59</second>
              <year>2024</year>
              <month>11</month>
              <day>30</day>
             </datetime>
            </property>
            <property name="minimumDateTime">
             <datetime>
              <hour>0</hour>
              <minute>0</minute>
              <second>0</second>
              <year>2024</year>
              <month>1</month>
              <day>1</day>
             </datetime>
            </property>
            <property name="maximumDate">
             <date>
              <year>2024</year>
              <month>11</month>
              <day>30</day>
             </date>
            </property>
            <property name="calendarPopup">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <property name="spacing">
           <number>12</number>
          </property>
          <item>
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>Дата окончания подачи документов</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDateEdit" name="endDate">
            <property name="dateTime">
             <datetime>
              <hour>0</hour>
              <minute>0</minute>
              <second>0</second>
              <year>2024</year>
              <month>1</month>
              <day>1</day>
             </datetime>
            </property>
            <property name="minimumDateTime">
             <datetime>
              <hour>0</hour>
              <minute>0</minute>
              <second>0</second>
              <year>2024</year>
              <month>1</month>
              <day>1</day>
             </datetime>
            </property>
            <property name="maximumDate">
             <date>
              <year>2024</year>
              <month>12</month>
              <day>31</day>
             </date>
            </property>
            <property name="minimumDate">
             <date>
              <year>2024</year>
              <month>1</month>
              <day>1</day>
             </date>
            </property>
            <property name="date">
             <date>
              <year>2024</year>
              <month>1</month>
              <day>1</day>
             </date>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QGridLayout" name="gridLayout">
          <property name="spacing">
           <number>4</number>
          </property>
          <item row="1" column="0">
           <widget class="QPushButton" name="saveBtn">
            <property name="text">
             <string>Сохранить</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QPushButton" name="deleteBtn">
            <property name="text">
             <string>Удалить</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QPushButton" name="createBtn">
            <property name="text">
             <string>Создать</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QPushButton" name="fillBtn">
            <property name="text">
             <string>Заполнить</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
